app/controllers/application_controller.rb

```diff
@@ -7,14 +7,6 @@ def favorite
     @favorite ||= Favorite.new(session[:favorite])
   end
 
-  def toggle_approval_status(pet_application)
-    if pet_application[:approval_status] == "unapproved"
-      "approved"
-    else
-      "unapproved"
-    end
-  end
-
```

Comment:
I moved this back into the controller because I ended up only using it in one class and it isn't good practice to inherit something only used by one class.

Reply:
Good call!